language-plutus-core/src/Language/PlutusCore/Constant/Apply.hs

```diff
@@ -268,3 +256,19 @@ applyStaticBuiltinName name args = do
                 (\b x y -> if b then x else y)
                 (runCostingFunThreeArguments $ paramIfThenElse params)
                 args
+
+builtinNameAritiesIgnoringTypes :: Array StaticBuiltinName Int
```

Comment:
Worth having a Haddock here.

Reply:
That's going to disappear in the alternative-ck-2 PR.